fix(connectors): remove retry logic when checking if app isAuthorized

tomiiide opened the pull request:
This PR fixes a user problem where all unisat based connectors calls `getAccounts()` repeatedly inside the `isAuthorized()`.

## Problem
1. `getAccounts()` triggers the connect pop up if the wallet is not connected, so this is not a great logic to check if a user is already connected.
2. we are calling `getAccounts` using a retry logic.

## Solution
1. Remove the `withRetry` logic, to remove the repeated calls.
2. Since we mange the connection state in storage in `connect` and `disconnect`, we don't need to make the redundant call to `getAccounts`, as it will be called again during a reconnection.

chybisov reviewed (commented):
Are we sure this is going to work for every wallet? Do we need to re-test every one of them?

tomiiide commented:
@chybisov  I have tested.
1. I updated the functions of the wallets that are directly affected (MagicEden, Bitget, and Okx) 
2. I removed the retry logic on all wallets as I think we don't need it and is excessive.

what do you think?
